styles/Hero.module.css

```diff
@@ -0,0 +1,82 @@
+.hero_container {
```

Comment:
The component is not showing correctly on mobile.

Try to avoid absolute positions to fix it. 

<img width="1912" alt="Screenshot 2023-03-01 at 12 49 27 AM" src="https://user-images.githubusercontent.com/11657599/222055979-54c85762-88ad-4c1a-9785-74dcbbdcef1d.png">